### PR TITLE
Created the FAQ page

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
https://github.com/pythonindia/inpycon2020-tasks/issues/67

CC: 
@palnabarun
@vipulgupta2048
@anistark
@RiteshAgrawal
@raukadah
@abhishekmishragithub
@Pradhvan
@rayandas
@inovizz
@vipulgupta2048
@sayanchowdhury 

Please give your reviews on the FAQ page and if any other question needs to be added. 

Also, I do not like the current font of the page. I am also creating another PR with the style changes on top of this one so that the changes can be discussed separately. #63

### CODE REVIEW

File: faq/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+You can submit your talk/workshop proposal oon our [CFP Page](https://in.pycon.org/cfp/2020/proposals/). The extended talks CFP will be open till August 19, and the workshop CFP will be open till August 30.
+
+### Q. I want to volunteer for the conference. What should I do?
+
+Check out our [Volunteer Blog](https://in.pycon.org/blog/2020/2020-call-for-volunteers.html) to know about the roles and responsibilities. You can also join our [Zulip channel](https://pyconindia.zulipchat.com/) and ping the respective workgroup leads to get started with volunteering.
+
+### How much time do I need to devote to the volunteering work if I choose to volunteer for PyCon India? 
+
+The do not demand any strict time requirements from our volunteers and value your every of contribution, no matter how small or large it is. You can volunteer on your own time and availability. But, ideally, one hour a day would be more than sufficient.
```
Comment (vipulgupta2048) [inline faq/index.md]:
```suggestion
There isn't any strict time requirement that needs to be fulfilled by the volunteers. We value each and every contribution, no matter how big or small it is. You can choose to volunteer on your own time and availability. But, ideally, one hour a day would be more than sufficient if you can keep up the commitment and do the work assigned to you.  
```

File: faq/index.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+### Q. I have bought the workshop ticket am I eligible to participate in DevSprint?
+
+**No**, the workshop tickets only include access to the specific workshops. You would have to buy the conference tickets to get access to the devsprints. 
+
+### Q. There is a change in my plan so I can’t attend the conference. Can I get a refund?
+
+**Yes**, the conference tickets can be cancelled through townscript. 
+
+{Procedure here}
```
Comment (vipulgupta2048) [inline faq/index.md]: @inovizz can help with this.

Comment (vipulgupta2048) [reply faq/index.md]: And the one below these.

**Review by vipulgupta2048 — CHANGES_REQUESTED** [review]
Looks solid on a quick run-through. Good work @Sc0rpi0n101

File: faq/index.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+**Yes**, the conference tickets can be transferred to another person via townscript. 
+
+{Procedure here}
+
+### Q. What does a contributor ticket mean?
+
+PyCon India is a conference run with contributions from volunteers and supporters from the community. Hence, as organizers, we provide multiple avenues for people to contribute to the event. People can contribute their time by volunteering in the various conference activities. People can also make a financial contribution by purchasing a contributor ticket.
+
+Contributor tickets help us meet the expenses of running the conference, and in the long run, can reduce the dependence on sponsors. People who purchase a contributor ticket will be credited in our blog, towards the end of the conference.
```
Comment (palnabarun) [inline faq/index.md]:
```suggestion
Contributor tickets help us meet the expenses of running the conference, and in the long run, reduce our dependence on sponsors. People who purchase a contributor ticket will be credited in our blog, towards the end of the conference.

```

File: faq/index.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+**Yes**, a separate ticket would have to be purchased for each workshop. The details for the workshop tickets will be updated soon. 
+
+### Q. Is there a different ticket for the Devsprints?
+
+**No**, the conference ticket includes complete access to the dev sprints. The ticket holders do not have to pay anything extra. But, the attendees will be asked to register for the dev sprints as that would help the conference with logistics. 
+
+### Q. I have bought the workshop ticket am I eligible to participate in DevSprint?
+
+**No**, the workshop tickets only include access to the specific workshops. You would have to buy the conference tickets to get access to the dev sprints.
```
Comment (palnabarun) [inline faq/index.md]: @inovizz Can you please have a look at this once?

Right now, we are getting some queries on Townscript about the separate Dev Sprint registrations. IMHO, we should make our terminology consistent across the ticket blog, FAQ and information on Townscript.

File: faq/index.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+**No**, PyCon India does not provide any certificates to the attendees/participants. The conference is a platform to share your experience, express your views, learn, and meet new people.
+
+
+## Ticketing FAQ
+
+---
+
+### Q. I am a speaker/volunteer at the conference. Do I need to buy a ticket?
+
+**Yes**. PyCon India is a completely volunteer-driven conference and we follow the principal _"Everyone Pays"_. Everyone from the speakers to volunteers, to even the organizers have to pay for their own tickets to attend the conference. Nobody earns anything from the conference, all the proceedings from the ticket sales go towards making the conference even better for everyone.
```
Comment (palnabarun) [inline faq/index.md]:
```suggestion
**Yes**. PyCon India is a conference built for the community and driven by the community. Everyone from the speakers to volunteers, to even the organizers have to pay for their own tickets to attend the conference. Nobody makes a dime during the whole event. All proceedings from the ticket sales go towards making the conference even better for everyone.

Here is a beautiful article by [Jesse Noller](https://twitter.com/jessenoller) about the same http://jessenoller.com/blog/2011/05/25/pycon-everybody-pays.

```

**Review by palnabarun — CHANGES_REQUESTED** [review]

File: faq/index.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+### Q. Is there a different ticket for Workshops?
+
+**Yes**, a separate ticket would have to be purchased for each workshop. The details for the workshop tickets will be updated soon. 
+
+### Q. Is there a different ticket for the Devsprints?
+
+**No**, the conference ticket includes complete access to the dev sprints. The ticket holders do not have to pay anything extra. But, the attendees will be asked to register for the dev sprints as that would help the conference with logistics. 
+
+### Q. I have bought the workshop ticket am I eligible to participate in DevSprint?
```
Comment (sayanchowdhury) [inline faq/index.md]:
```suggestion
### Q. I have bought the workshop ticket. Am I eligible to participate in DevSprints?
```

File: faq/index.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+**No**, the workshop tickets only include access to the specific workshops. You would have to buy the conference tickets to get access to the dev sprints. 
+
+### Q. There is a change in my plan so I can’t attend the conference. Can I get a refund?
```
Comment (sayanchowdhury) [inline faq/index.md]: No, you can only transfer tickets. We don't cancel tickets.

**Review by sayanchowdhury — CHANGES_REQUESTED** [review]

File: faq/index.md (hunk 7 above)
Comment (Sc0rpi0n101) [reply faq/index.md]: Alright I'll update it

File: faq/index.md (hunk 5 above)
Comment (Sc0rpi0n101) [reply faq/index.md]: How about this?

```suggestion
**Yes**. PyCon India is a conference built for the community, by the community. Everyone from the speakers to volunteers, to even the organizers have to pay for their own tickets to attend the conference. Nobody makes a dime during the whole event. All proceedings from the ticket sales go towards making the conference even better for everyone.

Here is a beautiful article by [Jesse Noller](https://twitter.com/jessenoller) about the same http://jessenoller.com/blog/2011/05/25/pycon-everybody-pays.
```

**Comment by Sc0rpi0n101** [comment]
ping @sayanchowdhury @palnabarun

**Review by palnabarun — APPROVED** [review]

**Comment by palnabarun** [comment]
@Sc0rpi0n101 Please squash the commits.

File: faq/index.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+
+### Q. Is there a different ticket for the Devsprints?
+
+**No**, the conference ticket includes complete access to the dev sprints. The ticket holders do not have to pay anything extra. But, the attendees will be asked to register for the dev sprints as that would help the conference with logistics. 
+
+### Q. I have bought the workshop ticket. Am I eligible to participate in DevSprints?
+
+**No**, the workshop tickets only include access to the specific workshops. You would have to buy the conference tickets to get access to the dev sprints. 
+
+### Q. There is a change in my plan so I can’t attend the conference. Can I get a refund?
```
Comment (sayanchowdhury) [inline faq/index.md]: We would need same for workshop as well

**Comment by Sc0rpi0n101** [comment]
Is your DevSprints comment resolved?

**Review by sayanchowdhury — CHANGES_REQUESTED** [review]

File: faq/index.md (hunk 8 above)
Comment (Sc0rpi0n101) [reply faq/index.md]: how about "Conference and/or workshops"?

Comment (Sc0rpi0n101) [reply faq/index.md]:
```suggestion
### Q. There is a change in my plan so I can’t attend the conference and/or workshops. Can I get a refund?

```

Comment (Sc0rpi0n101) [reply faq/index.md]: ping @sayanchowdhury

**Pull request merged**